.circleci/config.yml

```diff
@@ -35,6 +35,8 @@ jobs:
           name: Deploy
           command: |
             echo $PLOTLY_GA_CODE > ga_code.txt
+	    cat ga_code.txt
```

Comment:
If it is confidential, it is not a good idea to `cat` it here. if it is not, I dont see a reason why it needs to go to circle CI environment variables.